pkgs/development/python-modules/playwright/update.sh

```diff
@@ -4,16 +4,29 @@ set -euo pipefail
 
 root="$(dirname "$(readlink -f "$0")")"
 
-version=$(curl ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} -s https://api.github.com/repos/microsoft/playwright-python/releases/latest | jq -r '.tag_name | sub("^v"; "")')
-# Most of the time, this should be the latest stable release of the Node-based
-# Playwright version, but that isn't a guarantee, so this needs to be specified
-# as well:
-setup_py_url="https://github.com/microsoft/playwright-python/raw/v${version}/setup.py"
-driver_version=$(curl -Ls "$setup_py_url" | grep '^driver_version =' | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+')
```

Comment:
you should take driver version from setup.py

https://github.com/microsoft/playwright-python/blob/d9cdfbb1e178b6770625e9f857139aff77516af0/setup.py#L33